### PR TITLE
Revert "Update readme.md"

### PR DESCRIPTION
**Pull request by sach2212**
@NitkarshChourasia

### CODE REVIEW

**Comment by NitkarshChourasia** [comment]
Review option is at the side.

**Comment by sach2212** [comment]
> 
"Reviewers" this is the option on the right side but your user id is not getting

**Review by NitkarshChourasia — CHANGES_REQUESTED** [review]
CSS, assets, and front-end with position assigned will be done.
First list the requirements.
What features one expects?
Like,
- Online payment option
- Username login
- Profile image
- Cart
Etc...,

Instead of hard coding the process of designing and structuring,
let's use prebuilt frameworks like bootstrap, etc...,
then reiterate it.

**Comment by NitkarshChourasia** [comment]
Let's use Flask or Django for backend.
We will try to use MongoDB for backend.
For CSS, bootstrap will be good.
For HTML, you also try to figure out, frameworks.

Let's streamline the process and the workflow.
# Let's not reinvent the wheel.

@sach2212

**Review by NitkarshChourasia — COMMENTED** [review]
do update the following.
Use CHATGPT, to streamline the process.

**Comment by NitkarshChourasia** [comment]
>  Use CHATGPT, to streamline the process.

if needed.

**Comment by NitkarshChourasia** [comment]
@sach2212

**Comment by sach2212** [comment]
online payment we can have ,
also log in system with OTP (what you say..?),
and and yeah i agree with the CSS point  because designing is not the main goal we  have options for that,
Django is Python framework so lets totally work in the JS libraries (what you say)

**Comment by sach2212** [comment]
> > Use CHATGPT, to streamline the process.
> 
> if needed.

yes  lets do that

**Comment by NitkarshChourasia** [comment]
Okay.
So which framework we should be using?
REACT?
Also, you are right python based is django.
I was getting an internship, which they had project based **STACK** requirements.
They said, I should be knowing **REACT** also, **DJANGO** also, and **FLASK** also, and one more .js library.

If your goal is to in the backend and frontend or fullstack.
Then for now, I would say having a knowledge of these framework will be good.

Django or Flask if we are using now, will may ensure implementation of all the dependencies libraries and API(s),
but .js Stack is good, too.

Anything, would be fine.
Because learning Flask and Django taught me that if we learn implementation then backend approach also won't matter, just little here and there.

**Comment by NitkarshChourasia** [comment]
### So, tell me the STACK. @sach2212

**Comment by NitkarshChourasia** [comment]
> I was getting an internship, which they had project based STACK requirements.
>They said, I should be knowing REACT also, DJANGO also, and FLASK also, and one more .js library.

I was getting internship through a jack from Bangalore to US based startup firm.

**Comment by sach2212** [comment]
and lets start with this basics of ideas like what we have to add on the website.

 while building the project while actually being in the process lot of the thing will come in between and the ideas will click automatically so dive into the project and lets decide what is the role.

currently I don't have any idea in a front end frameworks but my focus is to learn  MERN STACK in this project (what you say)
main focus in the backend (you tell me what to do?
)

**Comment by sach2212** [comment]
> Okay. So which framework we should be using? REACT? Also, you are right python based is django. I was getting an internship, which they had project based **STACK** requirements. They said, I should be knowing **REACT** also, **DJANGO** also, and **FLASK** also, and one more .js library.
> 
> If your goal is to in the backend and frontend or fullstack. Then for now, I would say having a knowledge of these framework will be good.
> 
> Django or Flask if we are using now, will may ensure implementation of all the dependencies libraries and API(s), but .js Stack is good, too.
> 
> Anything, would be fine. Because learning Flask and Django taught me that if we learn implementation then backend approach also won't matter, just little here and there.

yeah let use react @NitkarshChourasia

**Comment by NitkarshChourasia** [comment]
### Okay, Sachin research the stack of your choice and then list them, also the database too. @sach2212

**Comment by sach2212** [comment]
**lets use react and mongo db**

**Comment by NitkarshChourasia** [comment]
### In .js see if React will be the right framework? or any other framework we need? in .js 
also research and list the desired frameworks for ... css and html, too.

**Comment by NitkarshChourasia** [comment]
> **lets use react and mongo db**

okay, let's use them, if they are the right stack.
Let's start learning then.

**Comment by sach2212** [comment]
for front end it will be fine and for backend mongo and node @NitkarshChourasia

**Comment by NitkarshChourasia** [comment]
### Stack are not chosen randomly, they are chosen based on requirements I guess, am I right? @sach2212

**Comment by sach2212** [comment]
> for front end it will be fine and for backend mongo and node @NitkarshChourasia

react

**Comment by NitkarshChourasia** [comment]
> > for front end it will be fine and for backend mongo and node @NitkarshChourasia
> 
> react

Okay.
CSS - bootstrap.
Html?
@sach2212

**Comment by sach2212** [comment]
> > > for front end it will be fine and for backend mongo and node @NitkarshChourasia
> > 
> > 
> > react
> 
> Okay. CSS - bootstrap. Html? @sach2212
REACT
yes @NitkarshChourasia 
and backend Mongodb

**Comment by NitkarshChourasia** [comment]
okay. @sach2212 take the lead.

**Comment by sach2212** [comment]
> ### Stack are not chosen randomly, they are chosen based on requirements I guess, am I right? @sach2212

I am totally new while using any LIB i have not use it before ,
so guide me for that like which will be the best so accordingly i have to work on my role

**Comment by sach2212** [comment]
in my opinion starts with JS libs onlu @NitkarshChourasia

**Comment by NitkarshChourasia** [comment]
okay.

**Comment by NitkarshChourasia** [comment]
@sach2212  so I am starting to learn .js REACT library and MONGODB library.
what do you say?

**Comment by NitkarshChourasia** [comment]
Please update all the requirements we would likely to have on README.md file. Please. ASAP. With Stack listed.
@sach2212

**Comment by sach2212** [comment]
yeah and what should  I do  @NitkarshChourasia

**Comment by sach2212** [comment]
> Please update all the requirements we would likely to have on README.md file. Please. ASAP. With Stack listed. @sach2212

yeah sure

**Comment by NitkarshChourasia** [comment]
> yeah and what should I do @NitkarshChourasia

With OTP, newsletter, etc...

**Comment by NitkarshChourasia** [comment]
@sach2212 for now learn good HTML framework and good CSS framework.

**Comment by NitkarshChourasia** [comment]
Bootstrap, In my opinion is good, you may research some other frameworks.
HTML, not looked into, you can research and use any good framework.

**Comment by sach2212** [comment]
and what I am  saying what ever i know  in a front ends  like in  html and CSS i will mention that i will make sure explaining that so that you will learn to 
  2. as a beginner programmer please help me to learn with  the backend as well  i am thinking while being in this project at the end of this project u should the  knowledge of front end and i should get the knowledge of backend like what you have done  in the backend  i need that much level of the understanding at the end of this project (vice verse) in short teach  me in this process 
  3.

**Comment by sach2212** [comment]
> Bootstrap, In my opinion is good, you may research some other frameworks. HTML, not looked into, you can research and use any good framework.

yeah I will research it  @NitkarshChourasia

**Comment by NitkarshChourasia** [comment]
@sach2212 don't worry let's just start, we will learn more then we can think of.
because if we already know, what we are going to learn, then we would have not even started it.
got it?

don't worry just code and code and code, and implement, everything will fall.
don't bifurcate only backend, yes we will learn ...**research about best way of pair programming** do share me the resource too, and we will try to implement it.
okay?
**git, github, pair programming** research them extensively also share me the resources, and we will try to implement it.

# Remember.
reject what is mere theory, accept what is being implementable.
think only through this mindset how can I implement it.
in short, 
become so pragmatic that...

@sach2212 

```
also, please use proper English, my boiii... no slang and things.
be formal.
```

**Comment by NitkarshChourasia** [comment]
@sach2212 
# Let me take front end fully.
CSS and it's library.
HTML and it's library.

**Comment by sach2212** [comment]
> @sach2212 don't worry let's just start, we will learn more then we can think of. because if we already know, what we are going to learn, then we would have not even started it. got it?
> 
> don't worry just code and code and code, and implement, everything will fall. don't bifurcate only backend, yes we will learn ...**research about best way of pair programming** do share me the resource too, and we will try to implement it. okay? **git, github, pair programming** research them extensively also share me the resources, and we will try to implement it.
> 
> # Remember.
> reject what is mere theory, accept what is being implementable. think only throught this mindset how can I implement it. in short, become so pragmatic that...
> 
> @sach2212

yeah i got you  @NitkarshChourasia 

and also i need to tell you in a day how much time we should give to this project 
in my perspective, I have edabit to do,  learning about gits, exploring more in JS, sometimes there will be freelancing projects not to much,

at the end what my point is time management is important

**Comment by NitkarshChourasia** [comment]
# Nothing exists as TIME MANAGEMENT.

Mantra is:
# EAT
# SLEEP
# CODE
# REPEAT....

yes, keeping time for leisure recreational activities, walking, gym, socializing, etc..,

**Comment by sach2212** [comment]
> # Nothing exists as TIME MANAGEMENT.
> Mantra is:
> 
> # EAT
> # SLEEP
> # CODE
> # REPEAT....
> yes, keeping time for leisure recreational activities, walking, gym, socializing, etc..,

yeah @NitkarshChourasia
